### ISSUE TITLE
Can we Extract data like Address from a text here I have no experience ho to get this done 

### ISSUE THREAD

**Issue by dheeraj-thedev** [knuppe/SharpNL#43]
I have a String that is coming in JSON format I used NER that is extracting Names, Dates, Companies etc successfully but i am stuck here to extract Address from that text please help me on this if is there any solution using NLP

Thanks

**Comment by knuppe** [comment]
Yes, it is possible, but I believe you will need a large data set to train a decent model. But to be honest, it's a lot of work to perform such a task...

A starting point is to read the documentation[documentation](https://opennlp.apache.org/docs/1.8.4/manual/opennlp.html#tools.namefind.training) that explains how to train NER models.
